container/rkt/helpers.go

```diff
@@ -41,24 +80,28 @@ func verifyName(name string) (bool, error) {
 */
 //TODO{sjpotter}: this currently only recognizes machined started pods, which actually doesn't help with k8s which uses them as systemd services, need a solution for both
 func parseName(name string) (*parsedName, error) {
```

Comment:
nit, can we break the TODO into multiple lines?
Also please add space after `//`